### ISSUE TITLE
`yarn build` fails Module not found: Can't resolve 'AccessibilityInfo' in 'E:\React Native\快速上手React Native开发\node_modules\react-native\Libraries\react-native'

### ISSUE THREAD

**Issue by cnscorpions** [dabbott/react-native-express#65]
environment info:
OS: win10 pro
Node: v12.13.0
Npm: 6.12.0
Yarn: 1.22.4
package.json as follows:
```
{
  "name": "react-native-express",
  "description": "Interactive examples, code samples, and tutorial projects for React Native",
  "version": "1.0.0",
  "author": "Devin Abbott <devinabbott@gmail.com> (http://github.com/dabbott)",
  "dependencies": {
    "@babel/core": "^7.0.0-0",
    "@mdx-js/mdx": "^1.0.20",
    "@mdx-js/react": "^1.0.0",
    "@mdx-js/tag": "^0.20.3",
    "@next/mdx": "^8.1.0",
    "@zeit/next-css": "^1.0.1",
    "babel-plugin-styled-components": "^1.10.1",
    "change-case": "^4.1.1",
    "disqus-react": "^1.0.5",
    "global": "^4.4.0",
    "graphql": "^0.13.0",
    "next": "^8.1.0",
    "next-images": "^1.1.2",
    "react": "^16.8.0",
    "react-dom": "^16.8.6",
    "react-ga": "^2.6.0",
    "react-helmet": "^5.2.1",
    "react-native": "0.55.4",
    "react-native-web-player": "^2.0.0-alpha.8",
    "react-redux": "^7.2.0",
    "redux": "^4.0.5",
    "styled-components": "^4.3.1"
  },
  "keywords": [],
  "license": "MIT",
  "scripts": {
    "dev": "next",
    "build": "next build",
    "start": "next start -p $PORT",
    "format": "prettier --write '**/*.js'",
    "git": "git add . && git commit -m",
    "postgit": "git pull && git push && git status",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "devDependencies": {
    "@mdx-js/loader": "^1.0.20",
    "eslint-plugin-graphql": "^3.0.3",
    "loader-utils": "^1.2.3",
    "prettier": "^1.13.7",
    "raw-loader": "^3.0.0",
    "webpack": "^4.0.0"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/gatsbyjs/gatsby-starter-default"
  }
}
```

1. yarn
2. yarn dev

both works well.

3. fix error, install react native as dependency for examples\files\todo_list\*.js

4.  yarn build
```
PS E:\React Native\快速上手React Native开发> yarn build
yarn run v1.22.4
$ next build
Creating an optimized production build ...

> Using external babel configuration
> Location: "E:\React Native\快速上手React Native开发\.babelrc"
Failed to compile.

./node_modules/react-native/Libraries/react-native/react-native-implementation.js
Module not found: Can't resolve 'AccessibilityInfo' in 'E:\React Native\快速上手React Native开发\node_modules\react-native\Libraries\react-native'

> Build error occurred
Error: > Build failed because of webpack errors
    at Object.build [as default] (E:\React Native\快速上手React Native开发\node_modules\next\dist\build\index.js:192:15)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Any help will be highly appreciated. :)

**Comment by dabbott** [comment]
What was the error before step 3? (what was the reason for installing `react-native`?)

The files in `examples/files` *should* be loaded as plain text files using the webpack `raw-loader`, so we shouldn't need to install `react-native` here. But if the error is that it's missing... I haven't tested on windows, but I wonder if the path regex I'm using doesn't work on windows?

https://github.com/dabbott/react-native-express/blob/6b2f550e96f36d4a490ab5d0dd4115d616a3a744/next.config.js#L13-L14

Maybe it should be `/examples(\/|\\)files(\/|\\).*.js/`

**Comment by cnscorpions** [comment]
@dabbott yes, it turns out `/examples\/files\/.*\.js$/` does not work in win10. And the new one works. Thank you. I'll close this issue. :)
